projects/igniteui-angular/src/lib/core/styles/components/tree/_tree-theme.scss

```diff
@@ -127,6 +127,8 @@
     %tree-display {
         display: block;
         z-index: 0;
+        height: 100%;
```

Comment:
@desig9stein we've reconsidered the whole approach. Could you please make an additional change and remove the whole height definition. Let the change be only adding `overflow-y: auto`